checker/tests/nullness/Issue752.java

```diff
@@ -0,0 +1,42 @@
+package a.b.c;
+
+import org.checkerframework.checker.nullness.qual.*;
+
+public class Issue752 {
+
+    Issue752 field = new Issue752();
+    static Issue752 staticField = new Issue752();
+
+    @RequiresNonNull("java.lang")
+    //:: error: (flowexpr.parse.error)
+    void method1() { }
+
+    @RequiresNonNull("java.lang.String")
```

Comment:
Shouldn't this issue an error?  The string might parse correctly as a type, but the Nullness Checker should not accept that as a legal argument to `@RequiresNonNull`.